pkg/epinio/models/epinio-resource.js

```diff
@@ -57,4 +59,37 @@ export default class EpinioResource extends Resource {
       throw epinioExceptionToErrorsArray(e);
     }
   }
+
+  async bulkRemove(items, opt = {}) {
+    if ( !opt.url ) {
+      opt.url = (this.links || {})['self'].replace(/\/[^\/]+$/, '?');
+    }
+    opt.method = 'delete';
+    opt.data = JSON.stringify({ unbind: false });
+
+    // Separates the resources by namespace
+    const _byNamespace = items.reduce((acc, cur) => {
+      const { namespace, name } = cur.meta;
+
+      if (!acc[namespace]) {
+        acc[namespace] = [];
+      }
+
+      acc[namespace].push(name);
+
+      return acc;
+    }, {});
+
+    const resPerNS = buildBulkLink(_byNamespace, this.type);
+
+    // Call the bulk remove for each namespace
+    await Promise.all(Object.entries(resPerNS).map(async([key, value]) => {
+      opt.url = `${ opt.url?.replace(/\/([^\/]*)\/([^\/]*)\/([^\/]*)\/([^\/]*)/, `/$1/$2/$3/${ key }`) }${ value }`;
```

Comment:
From what i can tell this just replaces the correct namespace with the correct namespace and then adds the query params. I don't think it's needed